website/docs/concepts/suspense.mdx

```diff
@@ -78,6 +78,17 @@ fn use_user() -> SuspensionResult<User> {
 }
 ```
 
+#### Note on implementing suspending hooks
+[`Suspension::new`](https://docs.rs/yew/latest/yew/suspense/struct.Suspension.html#method.new) returns 2 values: the suspension context itself, and a suspension handle.
+The latter is the one responsible for signaling when to re-render the suspended components, it provides 2 interchangable ways to do so:
+1. Calling its [`resume`](https://docs.rs/yew/latest/yew/suspense/struct.SuspensionHandle.html#method.resume) method.
+2. Dropping the handle.
+If the handle isn't stored for longer than the hook's scope, the suspended components will be constantly re-rendered as often as possible, thus hampering performance.
```

Comment:
I would highlight the infinite loop possibility in [`:::danger`](https://docusaurus.io/docs/markdown-features/admonitions). 

> #### Danger
> The handle must be stored until it's time to render the component. Not doing so could lead to infinite loops

You can word it better for it to go along with the rest of the section